test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -162,6 +164,123 @@ public static void Main()
         }
 
         [Fact]
+        public void Conflicts_are_resolved_when_publishing_a_portable_app()
+        {
+            Conflicts_are_resolved_when_publishing(false);
+        }
+
+        [Fact]
+        public void Conflicts_are_resolved_when_publishing_a_self_contained_app()
+        {
+            Conflicts_are_resolved_when_publishing(true);
+        }
+
+        void Conflicts_are_resolved_when_publishing(bool selfContained, [CallerMemberName] string callingMethod = "")
+        {
+            var targetFramework = "netcoreapp2.0";
+            var rid = selfContained ? EnvironmentInfo.GetCompatibleRid(targetFramework) : null;
```

Comment:
That build is working its way into the CLI now.  @dotnet-bot should be updating https://github.com/dotnet/cli/pull/6238 with the `1913` version soon.